getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -358,26 +362,26 @@ be assigned any other value.
 bool
 ^^^^
 
-The Boolean data type can only contain ``true`` or ``false``.
+The boolean data type can only contain ``true`` or ``false``.
```

Comment:
This is typically capitalized, as it's a proper name. Just a nitpick - I have no strong objection either way.

Reply:
According to [this Stack Exchange question](https://english.stackexchange.com/questions/4481/should-the-word-boolean-be-capitalized), capitalized "Boolean" is more common, so I'll go for that.